### PR TITLE
Add getEMHIRES pipeline

### PR DESCRIPTION
**Pull request by gperonato**

### CODE REVIEW

**Comment by melhk** [comment]
Why is there an empty file ` data-integration/getEEA.py`?

File: data-integration/datasets.csv [hunk 1]
```diff
@@ -14,13 +14,14 @@ Trends in patents: The data source is PATSTAT, the Worldwide Patent Statistical
 Private R&I investments: Data are estimated based on financial information from publicly available company statements and patent data from PATSTAT.
 Public (national) R&I investments: The International Energy Agency (IEA) statistics are the main source of data. For further information, see the methodology documentation.","""The SETIS estimations of private R&I are a metric aimed
 at enabling relative comparisons over time, rather than an accurate account of private
+<<<<<<< HEAD
```
Comment (melhk) [inline data-integration/datasets.csv]: I guess this line is to be removed

Comment (gperonato) [reply data-integration/datasets.csv]: Yes, thank you

**Comment by gperonato** [comment]
> Why is there an empty file ` data-integration/getEEA.py`?

It seemed to exist in my local branch... Anyway, I took the version from the master and pushed again

**Review by melhk — APPROVED** [review]

**Pull request merged**